content/docs/concepts/security/index.md

```diff
@@ -444,18 +444,23 @@ mutual TLS implementations.
 
 The `origins:` section defines authentication methods and associated parameters
 supported for origin authentication. Istio only supports JWT origin
-authentication. However, a policy can list multiple JWTs by different issuers.
+authentication. However, a policy can list multiple JWTs by different issuers and be enabled or disabled
+for specific paths. If all JWTs are disabled for a request path, authentication also passes as if
+there is none defined.
```

Comment:
However ... => You can specify allowed JWT issuers, and enable or disable JWT authentication for a specific path. 

"If all JWTs are disabled for a request path, authentication also passes as if there is none defined." This is confusing. You already said you can disable JWT authentication for a path, isn't that enough?

Reply:
That's added per @diemtvu's last comment. It makes it clear the default behavior when all JWT policies are disabled.

Reply:
I am ok if you want to keep this sentence. But I feel it is quite redundant.